synapse/models/person.py

```diff
@@ -90,6 +92,38 @@ def getBaseModels():
                     ('name:given', {'ptype': 'ps:tokn'}),
                 ]),
 
+                ('ps:contact', {'ptype': 'guid', 'doc': 'A collection of contact information in a single record'}, [
+
+                    ('org', {'ptype': 'ou:org', 'doc': 'The ou:org GUID which owns this contact'}),
+                    ('asof', {'ptype': 'time', 'doc': 'The time this contact was created or modified'}),
+                    ('person', {'ptype': 'ps:person', 'doc': 'The ps:person GUID which owns this contact'}),
+
+                    ('name', {'ptype': 'ps:name', 'doc': 'The person name listed for the contact'}),
+                    ('title', {'ptype': 'str:lwr', 'doc': 'The job/org title listed for this contact'}),
+                    ('photo', {'ptype': 'file:bytes', 'doc': 'The photo listed for this contact'}),
+
+                    ('orgname', {'ptype': 'str:lwr', 'doc': 'The listed org/company name for this contact'}),
+
+                    ('user', {'ptype': 'inet:user', 'doc': 'The username or handle for the contact'}),
+
+                    ('web:acct', {'ptype': 'inet:web:acct', 'doc': 'The social media account for this contact'}),
+
+                    ('dob', {'ptype': 'time', 'doc': 'The Date of Birth (DOB) for the contact'}),
+
+                    ('url', {'ptype': 'inet:url', 'doc': 'The home/main site for this contact'}),
+
+                    ('email', {'ptype': 'inet:email', 'doc': 'The main email address for this contact'}),
+                    ('email:work', {'ptype': 'inet:email', 'doc': 'The contact work email address'}),
```

Comment:
Given that a "contact" record might include both work and personal data (email, phone)...wondering if we should treat those the way we treat WHOIS records - where there is a unified "multi-contact" record (inet:whois:rec) but we also parse out individual "contacts" (inet:whois:contact). Would need to think this through a bit more but the concern is that "contact" info (like inet:web:acct info) potentially contains an arbitrary number of pieces of data (properties), so having to do things like email vs email:work or phone vs phone:work complicates pivoting across those properties.